src/shared/propTypes.ts

```diff
@@ -94,7 +94,7 @@ export const isValue = PropTypes.oneOfType([
 
 export const isViews = PropTypes.arrayOf(PropTypes.oneOf(allViews));
 
-export const isView: Requireable<View> = function isView(props, propName, componentName) {
+export const isView: Requireable<View> = function (props, propName, componentName) {
```

Comment:
We don't think you need a function name for that code.

Reply:
This makes sure we're getting the right error stack of something throws an error.

Reply:
I missed it, thanks